src/components/DateInfo/DateInfo.component.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-date-picker';
+import { PickerContainer, DateControls, DateBtn } from './DateInfo.styles';
+
+const DateInfo = (props) => {
+  const [pickerDate, setPickerDate] = useState(new Date());
+  const [current, setCurrent] = useState(false);
+  const isToday = new Date();
+
+  const handlePickerSelect = (newDate) => {
+    console.log(newDate);
```

Comment:
This is a non-blocker, but as a general rule we shouldn't be commiting console logs, this could expose data and app logic to third parties.